frontend/src/constants/ActionTypes.ts

```diff
@@ -0,0 +1,6 @@
+export enum actionTypes {
+    queryItem = "queryItem",
+    addItem = "addItem",
+    removeItem = "removeItem",
+    changeStoreOpt = "changeStoreOpt"
+}
```

Comment:
Could you add a newline to the end of this file? It doesn't affect functionality at all, but GitHub puts  big red circle when you don't end with a newline and red is a scary color lol